src/hotspot/share/classfile/vmIntrinsics.cpp

```diff
@@ -0,0 +1,862 @@
+/*
+ * Copyright (c) 1997, 2020, Oracle and/or its affiliates. All rights reserved.
```

Comment:
The copyright should start out with 2020 (with trailing comma) since this is a new file.